ripngd/ripng_cli.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 1998 Kunihiro Ishiguro
```

Comment:
How did Kunihiro do this work?

Reply:
The way I see it, it's still his code but in a different shape and form. That's why I decided to keep his copyright notice in the header of the new files (_ripng_cli.c_ and _ripng_northbound.c_).

On a separate note, I'll update this PR now to fix a typo (s/aggregate-addres/aggregate-address/).